app/views/layouts/_nav.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="prx-header-bar navbar navbar-dark align-items-stretch p-0 bg-primary-dark">
   <div class="container-fluid align-items-stretch pe-0">
     <header class="flex-grow-1 d-flex align-items-center">
-      <%= link_to image_tag("dovetail_logo.svg", width: 152, height: 30, alt: "Dovetail from PRX"), main_app.podcasts_path, class: "navbar-brand d-inline-flex" %>
+      <%= link_to image_tag("dovetail_logo.svg", width: 152, height: 30, alt: "Dovetail from PRX"), main_app.root_path, class: "navbar-brand d-inline-flex" %>
```

Comment:
hmm, I wonder if we should change the root+ podcasts path?
Like, in routes, change `root to: "base#entrypoint"` to root to: "podcasts#index" or something?